carbon-design-system/carbon
# feat(DataTable): add light prop

Closes #4120 

Add light prop to DataTable component for use on `$ui-01` container.
This is for the same reason as #4335 and #4347 

#### Changelog

**New**

- `light` prop in DataTable component

#### Testing / Reviewing

- Pull PR
- Make sure packages are built (`yarn build`)
- Run react storybook
```sh
cd packages/react
yarn storybook
```
- Activate light prop in knobs for DataTable. The DataTable rows background should change to the `$ui-02` equivalent color of the selected carbon theme.
